hack/generate/csv.sh

```diff
@@ -56,7 +56,7 @@ image "mt-broker-ingress__ingress"                  "${eventing}-mtbroker-ingres
 image "imc-controller__controller"                  "${eventing}-channel-controller"
 image "imc-dispatcher__dispatcher"                  "${eventing}-channel-dispatcher"
 
-image "v0.17.0-pingsource-cleanup__pingsource" "${eventing}-pingsource-cleanup"
+image "v0.17.0-pingsource-cleanup-eventing-$(metadata.get dependencies.eventing)__pingsource" "${eventing}-pingsource-cleanup"
```

Comment:
These changes are required due to https://github.com/knative/operator/commit/5b3bee26b2fa17c0213644d8c0a17f57fb523c69 (the OaaL operator is already based on 0.18 code). This makes jobnames more predictable and allows us to no longer mess with the manifest in the future, which is a win in my book.

Reply:
OK, so "simliar" job need same kinda "cosmetic" ?

Asking, b/c when we uodate to 0.18 that ping-cleanup job/image is gone